apps/dashboard/src/contract-ui/tabs/nfts/components/token-id.tsx

```diff
@@ -240,6 +245,26 @@ export const TokenIdPage: React.FC<TokenIdPageProps> = ({
                     </GridItem>
                   </>
                 )}
+                <GridItem colSpan={4}>
+                  <Heading size="label.md">Token URI</Heading>
+                </GridItem>
+                <GridItem
+                  colSpan={8}
+                  className="flex flex-row items-center gap-2"
+                >
+                  <AddressCopyButton
```

Comment:
Lets use the new shadcn component instead of chakra version ( `CopyTextButton` )